base_layer/wallet/src/output_manager_service/handle.rs

```diff
@@ -103,7 +103,7 @@ impl fmt::Display for OutputManagerRequest {
             GetCoinbaseTransaction(_) => write!(f, "GetCoinbaseTransaction"),
             GetPublicRewindKeys => write!(f, "GetPublicRewindKeys"),
             FeeEstimate(_) => write!(f, "FeeEstimate"),
-            RewindOutputs(_, _) => write!(f, "RewindAndImportOutputs"),
+            ScanForRecoverableOutputs(_, _) => write!(f, "ScanAndImportRecoverableOutputs"),
```

Comment:
just noticed these don't match, not sure it's an issue though

Reply:
fixed